CONTRIBUTING.md

```diff
@@ -199,5 +199,44 @@ $ yarn start --disable-host-check --public=$(echo ${server.dev-server} | sed -e
 ```
 
 ### Che server a.k.a WS master
+There is a [devfile](https://github.com/eclipse/che/blob/master/devfile.yaml) for development of Che server in Che.
+To build Che one may run a predefined build task from the devfile.
+
+Starting Che master requires some manual steps.
+Open a terminal in runtime container (`che-server-runtime`) and perform:
+ - First, set `CHE_HOME` environment variable with absolute path to parent folder of Che master's Tomcat.
+   It might look like `/projects/che/assembly/assembly-main/target/eclipse-che-*-SNAPSHOT/eclipse-che-*-SNAPSHOT`.
+ - Then set `CHE_HOST` with the endpoint of new Che master.
+   If using the [devfile](https://github.com/eclipse/che/blob/master/devfile.yaml) the endpoint is `che-dev` and already set.
```

Comment:
Could you change a hyperlink to the devfile on `[devfile](devfile.yaml)`?

Reply:
Will change